src/libs/actions/Report.js

```diff
@@ -338,23 +354,11 @@ function openReport(reportID, participantList = [], newReportObject = {}) {
             isOptimisticReport: true,
         };
 
-        // Change the method to set for new reports because it doesn't exist yet, is faster,
-        // and we need the data to be available when we navigate to the chat page
-        onyxData.optimisticData[0].onyxMethod = CONST.ONYX.METHOD.SET;
-
-        // Also create a report action so that the page isn't endlessly loading
-        onyxData.optimisticData[1] = {
-            onyxMethod: CONST.ONYX.METHOD.SET,
-            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
-            value: ReportUtils.buildOptimisticCreatedReportAction(newReportObject.ownerEmail),
-        };
+        // Add the createdReportActionID parameter to the API call
+        params.createdReportActionID = optimisticReportAction.reportActionID;
```

Comment:
This looks wrong – `buildOptimisticCreatedReportAction` returns an object with `0` as the key and the `reportAction` as the value. So I think you need this instead:

```suggestion
        params.createdReportActionID = optimisticReportAction[0].reportActionID;
```